tests/parsing/FilterHelperTest.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+from coalib import coala
+from coalib.parsing.FilterHelper import (
+    available_filters, InvalidFilterException)
+from tests.TestUtilities import (
+    bear_test_module,
+    execute_coala,
+    TEST_BEARS_COUNT,
+)
+
+
+class FilterHelperTest(unittest.TestCase):
+
+    def test_filter_for_exception(self):
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '-B', '--filter-by', 'some_filter',
+                'abc')
+            self.assertRaises(InvalidFilterException)
```

Comment:
that's not how this function works ;)

Anyway:
The issues states *not* to execute coala as it's too high-level for a unit-test. Please directly invoke `FilterHelper` methods.